oe_theme.theme

```diff
@@ -128,6 +129,7 @@ function oe_theme_preprocess_menu__main(array &$variables): void {
       'label' => $item['title'],
       'href' => $item['url'],
       'is_current' => $item['in_active_trail'],
+      'external' => ($item['url']->isExternal() && !strpos(UrlHelper::parse($item['url']->toString())['path'], 'europa.eu')),
```

Comment:
We need to centralize this logic, if we need to update/increase the domains that we consider "internal" it will be quite hard to do. On top of this, sites might want to define themselves what constitutes an "external" link

Reply:
Very good idea.